### PR TITLE
1-click-install-ThyraApp-from-thyra

### PR DESCRIPTION
**Pull request by B-Naoufal**
- Download ThyraApp from release page : [ThyraApp](https://github.com/massalabs/Thyra-Menu-Bar-App)
- Unzip it and launch it within the installation script

### CODE REVIEW

File: scripts/thyra-installer.py [hunk 1]
```diff
@@ -10,6 +10,8 @@
 # This file is to be bundled with pyinstaller in order to produce a .exe that can run on Windows without Python installed.
 
 THYRA_URL = "https://github.com/massalabs/thyra/releases/latest/download/thyra-server_windows_amd64"
+THYRA_APP = "https://github.com/massalabs/Thyra-Menu-Bar-App/releases/download/v0.0.1/ThyraApp_windows-amd64.exe"
```
Comment (gregLibert) [inline scripts/thyra-installer.py:13]: Why not using the latest version like for Thyra ?

File: scripts/thyra-installer.py [hunk 2]
```diff
@@ -10,6 +10,8 @@
...
+THYRA_APP_FILENAME = "ThyraApp_windows-amd64.exe"
 THYRA_FILENAME = "thyra-server.exe"
```
Comment (gregLibert) [inline scripts/thyra-installer.py:15]: Should be under THYRA_URL

File: scripts/thyra-installer.py [hunk 3]
```diff
@@ -131,7 +137,11 @@ def main():
         setupDNS()
     configureAcrylic()
     setupMkCerts()
+    THYRA_APP_PATH = os.path.join(USER_HOME_FOLDER, THYRA_APP_FILENAME)
     print("Thyra has been successfully installed! Executable is located at : " + USER_HOME_FOLDER)
+    if os.path.exists(THYRA_APP_PATH):
```
Comment (gregLibert) [inline scripts/thyra-installer.py:142]: Is it normal if it doesn't exist ? Is it possible ?

**Review by gregLibert — CHANGES_REQUESTED** [review]
I don't see any logic to make thyra start with next OS reboot.
@SlnPons don't you think that this is something mandatory ?

**Comment by B-Naoufal** [comment]
> I don't see any logic to make thyra start with next OS reboot. @SlnPons don't you think that this is something mandatory ?

will be covered in https://github.com/massalabs/Thyra-Menu-Bar-App/issues/7

**Review by gregLibert — APPROVED** [review]
:+1:

**Review by 0xMazout — APPROVED** [review]
Great

**Pull request merged**